aggie-coding-club/Rev-Registration
# Handle new instructional methods

## Description
Adds scraping, serialization, and fixes schedule generation handling for new instructional methods

## Rationale
Definitions for instructional methods:
![image](https://user-images.githubusercontent.com/28655462/97643202-12624e80-1a15-11eb-9d11-4f4fa31d9f9a.png)
Let me know if we should change any, it's very simple and doesn't involve making any migrations

## How to test
Try scraping courses for all terms and for `201111`, it should pass. I make an assertion that the instructional method is valid, so if anything is potentially not being handled we should be aware of it.

## Related tasks
Closes #431 and #433.
